### ISSUE TITLE
Be able to customize the font style (at least size) of code comments

### ISSUE THREAD

**Issue by DNin01** [ScratchAddons/ScratchAddons#6881]
### Why this would be helpful

@WorldLanguages[ commented](https://github.com/ScratchAddons/ScratchAddons/pull/5917#issuecomment-1779911648):
> From feedback: custom font size for editor comments. Seems like the missing piece of the puzzle. It's the only UI element in the Code tab that we don't provide a custom font size setting for.

The addon's "bold text" and "shadow under text" settings also don't work on comments.

### My suggestion

The settings in the `custom-block-text` addon should affect comments in the editor. Maybe not "shadow under text", though.

For those who would rather not apply a custom style to comments, we could also offer options for what text should be affected by these style settings.

**Comment by WorldLanguages** [comment]
It could also be a different setting, but I'm not sure if that's worth it.

**Comment by WorldLanguages** [comment]
Discussion #6959 lists some more parts of the editor where the text size isn't configurable.

**Comment by Secret-chest** [comment]
I don't think bold comments would look nice.
